indico/util/user.py

```diff
@@ -153,3 +155,25 @@ def wrapper(*args, **kwargs):
         return fn(*args, **kwargs)
 
     return wrapper
+
+
+def create_pending_user(user_data):
+    from indico.modules.users import User
+    from indico.modules.users.models.users import UserTitle
+
+    new_user = User()
+    new_user.email = user_data.get('email').lower()
+    new_user.first_name = user_data.get('firstName')
+    new_user.last_name = user_data.get('familyName')
+    new_user.title = int(UserTitle.get(re.sub(r'[^A-Za-z]', '', user_data.get('title').lower())))
+    new_user.is_pending = True
+
+    db.session.add(new_user)
+    db.session.flush()
+    return new_user
+
+
+def send_email_to_pending_contributors(email, event_name):
+    from indico.modules.auth.controllers import _send_confirmation
```

Comment:
Sure, however changing the name of this function could result in problems after merging one of my Pull Requests.